ios/Plugin/Plugin.swift

```diff
@@ -171,11 +255,23 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
             let accuracy = call.getDouble("accuracy") ?? 0.0
             let coordinate = CLLocationCoordinate2DMake(latitude, longitude)
             let location = CLLocation(coordinate: coordinate, altitude: -1, horizontalAccuracy: accuracy, verticalAccuracy: -1, timestamp: Date())
+            var accuracyLevel = RadarTrackingOptions.desiredAccuracy(for:"medium")
+            let beaconsTrackingOption = call.getBool("beacons") ?? false
+            var desiredAccuracy = call.getString("desiredAccuracy") ?? "medium"
+            desiredAccuracy = desiredAccuracy.lowercased()
+
+            if desiredAccuracy == "high" {
+                accuracyLevel = RadarTrackingOptions.desiredAccuracy(for:"high")
+            } else if desiredAccuracy == "medium" {
+                accuracyLevel = RadarTrackingOptions.desiredAccuracy(for:"medium")
+            } else if desiredAccuracy == "low" {
+                accuracyLevel = RadarTrackingOptions.desiredAccuracy(for:"low")
+            }
```

Comment:
Let's reject if `desiredAccuracy` is not a valid value

Reply:
same here, Typescript pre check null value, so Java/Swift, it can't be `null` value